src/main/java/com/powsybl/openloadflow/network/util/GenerationActionPowerDistributionStep.java

```diff
@@ -23,6 +25,29 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(GenerationActionPowerDistributionStep.class);
 
+    Map<LfGenerator, Double> initialValues = new HashMap<>();
+
+    public GenerationActionPowerDistributionStep() {
+        super();
+    }
+
+    public GenerationActionPowerDistributionStep(List<ParticipatingElement> elementsToMemorize) {
+        this();
+        initialValues = elementsToMemorize.stream()
```

Comment:
For me this is a bad design, this utility class should not have a state

Reply:
We have started to work on the network to avoid that: https://github.com/powsybl/powsybl-open-loadflow/pull/211
This aim is to have the same mechanism for security analysis and for sensitivity analysis. Please just tell me if we can follow this solution or not.